org.laptop.TurtleArtActivity.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.laptop.TurtleArtActivity",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.34",
+    "runtime-version": "42",
```

Comment:
43 is likely to be released in the next few hours!

Reply:
Err ...